sys/include/net/ng_ieee802154.h

```diff
@@ -58,6 +60,15 @@ extern "C" {
 #define NG_IEEE802154_FCF_SRC_ADDR_LONG     (0xc0)
 /** @} */
 
+/**
+ * @brief Data type to represent an EUI-64.
+ */
+typedef union __attribute__((packed)) {
+    uint64_t uint64;    /**< represented as 64 bit value */
+    uint8_t uint8[8];   /**< split into 8 8-bit words. */
+    uint16_t uint16[4]; /**< split into 4 16-bit words. */
```

Comment:
I'd prefer some sort of enforced byteorder here... Either `le_uintX_t` (to keep the spirit of IEEE 802.15.4) or `network_uintX_t` (to be compatible with everything else).